### ISSUE TITLE
ACL support

### ISSUE THREAD

**Issue by adamkrone** [AcalephStorage/consul-alerts#44]
Are there any plans on supporting Consul's ACL system?

https://consul.io/docs/internals/acl.html

This could be configured with a flag like `--consul-acl-token=some-token-here`

**Comment by atbell** [comment]
+1

**Comment by provonet** [comment]
+1

**Comment by darkcrux** [comment]
+1 pretty useful stuff.

**Comment by onnimonni** [comment]
+1 this is important

**Comment by sean-** [comment]
I think this can be closed.  This is exactly what was implemented for 0.3.1:

```sh
#!/bin/sh --

exec 2>&1
exec \
/usr/bin/env -i \
/usr/local/sbin/chpst \
        -e ./env \
        -u nobody:nobody \
./bin/consul-alerts start \
        --consul-dc=snc1 \
        --watch-checks \
        --watch-events \
        --consul-acl-token=${TOKEN_ID}
```
